model/src/pyrenew/metaclass.py

```diff
@@ -21,6 +21,36 @@
 from pyrenew.transformation import Transform
 
 
+def _assert_type_error(arg_name: str, value, expected_type) -> None:
```

Comment:
I'd call this just `_assert_type` rather than `_assert_type_error` since it passes without error if the type is as expected 
 
```suggestion
def _assert_type(arg_name: str, value, expected_type) -> None:
```

Reply:
changed in https://github.com/CDCgov/multisignal-epi-inference/pull/351/commits/7ad67f8b02315ffee008fa6d8bd44aae7867862a